src/App.js

```diff
@@ -1,24 +1,10 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator.js';
 
 function App() {
```

Comment:
- [ ] Let's make sure you are using class based component as requested in the project guidelines.